doc/radosgw/qat-accel.rst

```diff
@@ -142,14 +167,65 @@ Configuration
 
       export QAT_SECTION_NAME=CEPH
 
-   Next, edit the Ceph configuration file to enable QAT support for compression::
+   **For in-tree QAT**
+
+   For Intree QAT, Make sure your system meets `QATlib System Requirements`_ .
+   And QATlib can be installed from pre-built packages or source code,
+   please see `QATlib Installation`_ . After ready the qatlib, you can run
```

Comment:
s/After ready the qatlib/When QATlib has been installed/

Reply:
I can make this alteration later today. No action is necessary, @hualongfeng.

Reply:
Got it. Thanks

Reply:
Thank you for this update, @hualongfeng.

Do these changes apply to Squid, Reef, and Quincy?

Reply:
Apply to Squid, not to Reef, and Quincy.

Reply:
Roger that. Thanks, @hualongfeng.